server/character_data.py

```diff
@@ -335,6 +369,19 @@ def get_character_calendar(character_id, current_user=None):
     return {'info': calendar_data}
 
 
+@cachetools.cached(cachetools.TTLCache(maxsize=1000, ttl=SECONDS_TO_CACHE))
+def get_character_calendar_event(character_id, event_id, current_user=None):
+    character = Character.get(character_id)
+    character_application_access_check(current_user, character)
+    event_data = character.get_op(
+        'get_characters_character_id_calendar_event_id',
+        character_id=character_id,
+        event_id=event_id,
+    )
+    event_data['owner_name'] = get_details_for_id(event_data['owner_id'])['name']
+    return {'info': event_data}
+
+
 def get_transaction_party(party_id):
```

Comment:
I need to re-write the helper API to get all parties in parallel, so this function and the helper function it calls now in models.eve will be removed by me in a future PR. Also though, please don't put helper functions that aren't methods on models into models - we should put them in new modules within root or other new subdirectories.

Reply:
ok cool